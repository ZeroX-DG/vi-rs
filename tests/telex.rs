mod shared;

fn snapshot_transform(lines: &str) -> String {
    shared::transform_lines(lines, |word| {
        let mut trasformed_word = String::new();
        vi::telex::transform_buffer(word.chars(), &mut trasformed_word);
        trasformed_word
    })
}

macro_rules! gen_test_telex {
    ($name:tt, $path: tt) => {
        gen_test!(snapshot_transform, $name, $path);
    };
}
gen_test_telex!(simple_telex, "../testdata/input/simple_telex.txt");
gen_test_telex!(
    non_vietnamese_telex,
    "../testdata/input/non_vietnamese_telex.txt"
);
gen_test_telex!(
    bogo_engine_telex,
    "../testdata/input/bogo_engine_telex.txt"
);