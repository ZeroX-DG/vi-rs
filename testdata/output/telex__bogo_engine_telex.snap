---
source: tests/telex.rs
expression: snapshot_transform(contents)
info:
  input_file: "../testdata/input/bogo_engine_telex.txt"
---
chửi
chửi
chửi
chửi
chửi
chửi
bóp
bóp
bọp
bọp
bốc
bốc
bốc
bốc
bốc
bốc
bộc
bộc
bộc
bộc
bộc
bộc
khuâng
khuâng
khuâng
biêng
biêng
biêng
biếng
biếng
biếng
biếng
biếng
biếng
biếng
biếng
biếng
biếng
biếng
biếng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
chiềng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
khiễng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
kiểng
liệng
liệng
liệng
liệng
liệng
liệng
liệng
liệng
liệng
liệng
liệng
liệng
duyệt
duyệt
duyệt
duyệt
duyệt
duyệt
huyết
huyết
huyết
huyết
huyết
huyết
băng
băng
băng
bắng
bắng
bắng
bắng
bắng
bắng
bắng
bắng
bắng
bắng
bắng
bắng
bằng
bằng
bằng
bằng
bằng
bằng
bằng
bằng
bằng
bằng
bằng
bằng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bẵng
bặng
bặng
bặng
bặng
bặng
bặng
bặng
bặng
bặng
bặng
bặng
bặng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
chẳng
bèn
bèn
bén
bén
bẹn
bẹn
bẽn
bẽn
chẻn
chẻn
đen
đen
đen
bù
bú
bụ
bủ
cũ
đu
đu
hoăng
hoăng
hoăng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
hoẵng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
khoắng
loằng
loằng
loằng
loằng
loằng
loằng
loằng
loằng
loằng
loằng
loằng
loằng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
oẳng
bước
bước
bước
bước
bước
bước
bước
bước
bước
bước
bước
bước
chược
chược
chược
chược
chược
chược
chược
chược
chược
chược
chược
chược
yếm
yếm
yếm
yếm
yếm
yếm
yểm
yểm
yểm
yểm
yểm
yểm
khuây
khuây
khuấy
khuấy
khuấy
khuấy
khuấy
khuấy
nguẩy
nguẩy
nguẩy
nguẩy
nguẩy
nguẩy
bìm
bìm
bím
bím
bỉm
bỉm
hĩm
hĩm
lịm
lịm
quyên
quyên
quyến
quyến
quyến
quyến
quyến
quyến
quyền
quyền
quyền
quyền
quyền
quyền
quyển
quyển
quyển
quyển
quyển
quyển
quyện
quyện
quyện
quyện
quyện
quyện
bép
bép
bẹp
bẹp
bìu
bìu
bíu
bíu
bĩu
bĩu
bỉu
bỉu
bịu
bịu
khuỵu
khuỵu
khuỷu
khuỷu
bôi
bôi
bối
bối
bối
bối
bối
bối
bồi
bồi
bồi
bồi
bồi
bồi
bổi
bổi
bổi
bổi
bổi
bổi
bội
bội
bội
bội
bội
bội
cỗi
cỗi
cỗi
cỗi
cỗi
cỗi
huê
huề
huề
huệ
huệ
nhuế
nhuế
thuể
thuể
bưng
bưng
bưng
bứng
bứng
bứng
bứng
bứng
bứng
bứng
bứng
bứng
bứng
bứng
bứng
bừng
bừng
bừng
bừng
bừng
bừng
bừng
bừng
bừng
bừng
bừng
bừng
bửng
bửng
bửng
bửng
bửng
bửng
bửng
bửng
bửng
bửng
bửng
bửng
chững
chững
chững
chững
chững
chững
chững
chững
chững
chững
chững
chững
dựng
dựng
dựng
dựng
dựng
dựng
dựng
dựng
dựng
dựng
dựng
dựng
buốt
buốt
buốt
buốt
buốt
buốt
buột
buột
buột
buột
buột
buột
bốt
bốt
bốt
bốt
bốt
bốt
bột
bột
bột
bột
bột
bột
chuẩn
chuẩn
chuẩn
chuẩn
chuẩn
chuẩn
huân
huân
huấn
huấn
huấn
huấn
huấn
huấn
luận
luận
luận
luận
luận
luận
nhuần
nhuần
nhuần
nhuần
nhuần
nhuần
thuẫn
thuẫn
thuẫn
thuẫn
thuẫn
thuẫn
bông
bông
bông
bống
bống
bống
bống
bống
bống
bống
bống
bống
bống
bống
bống
bồng
bồng
bồng
bồng
bồng
bồng
bồng
bồng
bồng
bồng
bồng
bồng
bổng
bổng
bổng
bổng
bổng
bổng
bổng
bổng
bổng
bổng
bổng
bổng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bỗng
bộng
bộng
bộng
bộng
bộng
bộng
bộng
bộng
bộng
bộng
bộng
bộng
biếu
biếu
biếu
biếu
biếu
biếu
biểu
biểu
biểu
biểu
biểu
biểu
chiêu
chiêu
chiều
chiều
chiều
chiều
chiều
chiều
diễu
diễu
diễu
diễu
diễu
diễu
diệu
diệu
diệu
diệu
diệu
diệu
choăn
choăn
hoẳn
hoẳn
hoẳn
hoẳn
hoẳn
hoẳn
khoắn
khoắn
khoắn
khoắn
khoắn
khoắn
ngoằn
ngoằn
ngoằn
ngoằn
ngoằn
ngoằn
bò
bó
bõ
bọ
bỏ
đo
đo
hoăm
hoăm
hoắm
hoắm
hoắm
hoắm
hoắm
hoắm
khoằm
khoằm
khoằm
khoằm
khoằm
khoằm
bương
bương
bương
bương
bương
bương
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
bướng
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chường
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưởng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
chưỡng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
dượng
choạc
choạc
hoác
hoác
bàu
bàu
báu
báu
bạu
bạu
bảu
bảu
đau
đau
đau
bách
bách
bách
bạch
bạch
bạch
bục
bục
chúc
chúc
khuất
khuất
khuất
khuất
khuất
khuất
luật
luật
luật
luật
luật
luật
bịn
bịn
chín
chín
chĩn
chĩn
chỉn
chỉn
gìn
gìn
đin
đin
đin
biếc
biếc
biếc
biếc
biếc
biếc
diệc
diệc
diệc
diệc
diệc
diệc
bắt
bắt
bắt
bắt
bắt
bắt
bặt
bặt
bặt
bặt
bặt
bặt
bè
bé
bẹ
bẻ
bẽ
đe
đe
biết
biết
biết
biết
biết
biết
biệt
biệt
biệt
biệt
biệt
biệt
coóc
coóc
bòi
bòi
bói
bói
bỏi
bỏi
chọi
chọi
cõi
cõi
đoi
đoi
đoi
bôm
bôm
bồm
bồm
bồm
bồm
bồm
bồm
chỗm
chỗm
chỗm
chỗm
chỗm
chỗm
cốm
cốm
cốm
cốm
cốm
cốm
cộm
cộm
cộm
cộm
cộm
cộm
hổm
hổm
hổm
hổm
hổm
hổm
bấm
bấm
bấm
bấm
bấm
bấm
bầm
bầm
bầm
bầm
bầm
bầm
bẩm
bẩm
bẩm
bẩm
bẩm
bẩm
bẫm
bẫm
bẫm
bẫm
bẫm
bẫm
bậm
bậm
bậm
bậm
bậm
bậm
châm
châm
choèn
choèn
hoẻn
hoẻn
băn
băn
bắn
bắn
bắn
bắn
bắn
bắn
bằn
bằn
bằn
bằn
bằn
bằn
bẳn
bẳn
bẳn
bẳn
bẳn
bẳn
bặn
bặn
bặn
bặn
bặn
bặn
chẵn
chẵn
chẵn
chẵn
chẵn
chẵn
buồm
buồm
buồm
buồm
buồm
buồm
chuôm
chuôm
cuỗm
cuỗm
cuỗm
cuỗm
cuỗm
cuỗm
luộm
luộm
luộm
luộm
luộm
luộm
nhuốm
nhuốm
nhuốm
nhuốm
nhuốm
nhuốm
búp
búp
bụp
bụp
bươm
bươm
bươm
bươm
bướm
bướm
bướm
bướm
bướm
bướm
bướm
bướm
bướm
bướm
bướm
bướm
chườm
chườm
chườm
chườm
chườm
chườm
chườm
chườm
chườm
chườm
chườm
chườm
gượm
gượm
gượm
gượm
gượm
gượm
gượm
gượm
gượm
gượm
gượm
gượm
bàng
bàng
bàng
báng
báng
báng
bảng
bảng
bảng
chạng
chạng
chạng
hãng
hãng
hãng
đang
đang
đang
đang
chiếp
chiếp
chiếp
chiếp
chiếp
chiếp
diệp
diệp
diệp
diệp
diệp
diệp
khoèo
khoèo
ngoéo
ngoéo
yêu
yêu
yếu
yếu
yếu
yếu
yếu
yếu
yểu
yểu
yểu
yểu
yểu
yểu
bịp
bịp
chíp
chíp
chóa
chóa
dọa
dọa
hòa
hòa
hỏa
hỏa
ngõa
ngõa
bấp
bấp
bấp
bấp
bấp
bấp
bập
bập
bập
bập
bập
bập
béng
béng
béng
kẻng
kẻng
kẻng
mèng
mèng
mèng
hoáy
hoáy
ngoảy
ngoảy
ký
kỳ
kỵ
kỷ
kỹ
bô
bố
bố
bồ
bồ
bổ
bổ
bỗ
bỗ
bộ
bộ
choài
choài
choái
choái
choãi
choãi
hoại
hoại
hoải
hoải
chuyên
chuyên
chuyến
chuyến
chuyến
chuyến
chuyến
chuyến
chuyền
chuyền
chuyền
chuyền
chuyền
chuyền
chuyển
chuyển
chuyển
chuyển
chuyển
chuyển
chuyện
chuyện
chuyện
chuyện
chuyện
chuyện
huyễn
huyễn
huyễn
huyễn
huyễn
huyễn
bôn
bôn
bốn
bốn
bốn
bốn
bốn
bốn
bồn
bồn
bồn
bồn
bồn
bồn
bổn
bổn
bổn
bổn
bổn
bổn
bộn
bộn
bộn
bộn
bộn
bộn
hỗn
hỗn
hỗn
hỗn
hỗn
hỗn
hoạt
hoạt
khoát
khoát
choạp
choạp
buộc
buộc
buộc
buộc
buộc
buộc
chuốc
chuốc
chuốc
chuốc
chuốc
chuốc
chóe
chóe
chọe
chọe
hòe
hòe
khỏe
khỏe
tõe
tõe
bên
bên
bến
bến
bến
bến
bến
bến
bền
bền
bền
bền
bền
bền
bện
bện
bện
bện
bện
bện
hển
hển
hển
hển
hển
hển
phễn
phễn
phễn
phễn
phễn
phễn
giêng
giêng
giêng
giếng
giếng
giếng
giếng
giếng
giếng
giếng
giếng
giếng
giếng
giếng
giếng
giềng
giềng
giềng
giềng
giềng
giềng
giềng
giềng
giềng
giềng
giềng
giềng
buồi
buồi
buồi
buồi
buồi
buồi
buổi
buổi
buổi
buổi
buổi
buổi
chuôi
chuôi
chuối
chuối
chuối
chuối
chuối
chuối
chuỗi
chuỗi
chuỗi
chuỗi
chuỗi
chuỗi
chuội
chuội
chuội
chuội
chuội
chuội
bớt
bớt
bớt
bớt
bớt
bớt
bợt
bợt
bợt
bợt
bợt
bợt
bâng
bâng
bâng
bầng
bầng
bầng
bầng
bầng
bầng
bầng
bầng
bầng
bầng
bầng
bầng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
cẫng
hấng
hấng
hấng
hấng
hấng
hấng
hấng
hấng
hấng
hấng
hấng
hấng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
hẩng
nậng
nậng
nậng
nậng
nậng
nậng
nậng
nậng
nậng
nậng
nậng
nậng
bùi
bùi
búi
búi
bụi
bụi
chũi
chũi
củi
củi
đui
đui
đui
bây
bây
bấy
bấy
bấy
bấy
bấy
bấy
bầy
bầy
bầy
bầy
bầy
bầy
bẩy
bẩy
bẩy
bẩy
bẩy
bẩy
bẫy
bẫy
bẫy
bẫy
bẫy
bẫy
bậy
bậy
bậy
bậy
bậy
bậy
ngoàm
ngoàm
ngoạm
ngoạm
choán
choán
doãn
doãn
hoàn
hoàn
hoạn
hoạn
khoản
khoản
đoan
đoan
đoan
đoan
chùy
chùy
húy
húy
hủy
hủy
lũy
lũy
lụy
lụy
bắc
bắc
bắc
bắc
bắc
bắc
chặc
chặc
chặc
chặc
chặc
chặc
biếm
biếm
biếm
biếm
biếm
biếm
chiêm
chiêm
diềm
diềm
diềm
diềm
diềm
diềm
diễm
diễm
diễm
diễm
diễm
diễm
hiểm
hiểm
hiểm
hiểm
hiểm
hiểm
kiệm
kiệm
kiệm
kiệm
kiệm
kiệm
bơm
bơm
bờm
bờm
bờm
bờm
bờm
bờm
bợm
bợm
bợm
bợm
bợm
bợm
chớm
chớm
chớm
chớm
chớm
chớm
chởm
chởm
chởm
chởm
chởm
chởm
cỡm
cỡm
cỡm
cỡm
cỡm
cỡm
buông
buông
buông
buồng
buồng
buồng
buồng
buồng
buồng
buồng
buồng
buồng
buồng
buồng
buồng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
chuộng
cuống
cuống
cuống
cuống
cuống
cuống
cuống
cuống
cuống
cuống
cuống
cuống
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
luỗng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
thuổng
quýt
quýt
quỵt
quỵt
bê
bế
bế
bề
bề
bể
bể
bễ
bễ
bệ
bệ
bưa
bưa
bừa
bừa
bừa
bừa
bừa
bừa
bửa
bửa
bửa
bửa
bửa
bửa
bữa
bữa
bữa
bữa
bữa
bữa
bựa
bựa
bựa
bựa
bựa
bựa
chứa
chứa
chứa
chứa
chứa
chứa
bèo
bèo
béo
béo
bẹo
bẹo
bẻo
bẻo
bẽo
bẽo
đeo
đeo
đeo
bư
bứ
bứ
bự
bự
chừ
chừ
chữ
chữ
cử
cử
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuếch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
chuệch
bếp
bếp
bếp
bếp
bếp
bếp
rệp
rệp
rệp
rệp
rệp
rệp
bót
bót
bọt
bọt
biên
biên
biến
biến
biến
biến
biến
biến
biền
biền
biền
biền
biền
biền
biển
biển
biển
biển
biển
biển
biện
biện
biện
biện
biện
biện
diễn
diễn
diễn
diễn
diễn
diễn
bình
bình
bình
bính
bính
bính
bĩnh
bĩnh
bĩnh
bỉnh
bỉnh
bỉnh
bịnh
bịnh
bịnh
đinh
đinh
đinh
đinh
bào
bào
báo
báo
bão
bão
bạo
bạo
bảo
bảo
đao
đao
đao
bùn
bùn
bún
bún
bủn
bủn
chũn
chũn
lụn
lụn
đun
đun
đun
bành
bành
bành
bánh
bánh
bánh
bạnh
bạnh
bạnh
bảnh
bảnh
bảnh
hãnh
hãnh
hãnh
đanh
đanh
đanh
đanh
khoào
khoào
ngoáo
ngoáo
choẹt
choẹt
hoét
hoét
bày
bày
bảy
bảy
cháy
cháy
chạy
chạy
dãy
dãy
đay
đay
đay
bàn
bàn
bán
bán
bạn
bạn
bản
bản
giãn
giãn
đan
đan
đan
bút
bút
bụt
bụt
bươu
bươu
bươu
bươu
bướu
bướu
bướu
bướu
bướu
bướu
bướu
bướu
bướu
bướu
bướu
bướu
phượu
phượu
phượu
phượu
phượu
phượu
phượu
phượu
phượu
phượu
phượu
phượu
tườu
tườu
tườu
tườu
tườu
tườu
tườu
tườu
tườu
tườu
tườu
tườu
bất
bất
bất
bất
bất
bất
bật
bật
bật
bật
bật
bật
bấc
bấc
bấc
bấc
bấc
bấc
bậc
bậc
bậc
bậc
bậc
bậc
băm
băm
bằm
bằm
bằm
bằm
bằm
bằm
bặm
bặm
bặm
bặm
bặm
bặm
chắm
chắm
chắm
chắm
chắm
chắm
khẳm
khẳm
khẳm
khẳm
khẳm
khẳm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
đẵm
bát
bát
bạt
bạt
choắt
choắt
choắt
choắt
choắt
choắt
ngoặt
ngoặt
ngoặt
ngoặt
ngoặt
ngoặt
bênh
bênh
bênh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bềnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
bệnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chếnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chểnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
chễnh
quýnh
quýnh
quýnh
quỳnh
quỳnh
quỳnh
quỷnh
quỷnh
quỷnh
chêm
chêm
chềm
chềm
chềm
chềm
chềm
chềm
chễm
chễm
chễm
chễm
chễm
chễm
giếm
giếm
giếm
giếm
giếm
giếm
nệm
nệm
nệm
nệm
nệm
nệm
huých
huých
huých
huỵch
huỵch
huỵch
doành
doành
doành
hoạnh
hoạnh
hoạnh
hoảnh
hoảnh
hoảnh
bươi
bươi
bươi
bươi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
bưởi
cưới
cưới
cưới
cưới
cưới
cưới
cưới
cưới
cưới
cưới
cưới
cưới
cười
cười
cười
cười
cười
cười
cười
cười
cười
cười
cười
cười
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
cưỡi
rượi
rượi
rượi
rượi
rượi
rượi
rượi
rượi
rượi
rượi
rượi
rượi
bươn
bươn
bươn
bươn
lườn
lườn
lườn
lườn
lườn
lườn
lườn
lườn
lườn
lườn
lườn
lườn
lượn
lượn
lượn
lượn
lượn
lượn
lượn
lượn
lượn
lượn
lượn
lượn
mướn
mướn
mướn
mướn
mướn
mướn
mướn
mướn
mướn
mướn
mướn
mướn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
phưỡn
bưu
bưu
bửu
bửu
bửu
bửu
bửu
bửu
cứu
cứu
cứu
cứu
cứu
cứu
cừu
cừu
cừu
cừu
cừu
cừu
cữu
cữu
cữu
cữu
cữu
cữu
cựu
cựu
cựu
cựu
cựu
cựu
bơn
bơn
bờn
bờn
bờn
bờn
bờn
bờn
bỡn
bỡn
bỡn
bỡn
bỡn
bỡn
bợn
bợn
bợn
bợn
bợn
bợn
cớn
cớn
cớn
cớn
cớn
cớn
lởn
lởn
lởn
lởn
lởn
lởn
quyết
quyết
quyết
quyết
quyết
quyết
quyệt
quyệt
quyệt
quyệt
quyệt
quyệt
bác
bác
bạc
bạc
bượt
bượt
bượt
bượt
bượt
bượt
bượt
bượt
bượt
bượt
bượt
bượt
khướt
khướt
khướt
khướt
khướt
khướt
khướt
khướt
khướt
khướt
khướt
khướt
yêng
yêng
yêng
yểng
yểng
yểng
yểng
yểng
yểng
yểng
yểng
yểng
yểng
yểng
yểng
huơ
thuở
thuở
bâu
bâu
bấu
bấu
bấu
bấu
bấu
bấu
bầu
bầu
bầu
bầu
bầu
bầu
bậu
bậu
bậu
bậu
bậu
bậu
chẫu
chẫu
chẫu
chẫu
chẫu
chẫu
cẩu
cẩu
cẩu
cẩu
cẩu
cẩu
hừm
hừm
hừm
hừm
hừm
hừm
boóng
boóng
boóng
choòng
choòng
choòng
bài
bài
bái
bái
bãi
bãi
bại
bại
bải
bải
đai
đai
đai
bòm
bòm
bõm
bõm
bỏm
bỏm
cọm
cọm
dóm
dóm
đom
đom
đom
bức
bức
bức
bức
bức
bức
bực
bực
bực
bực
bực
bực
bứt
bứt
bứt
bứt
bứt
bứt
bựt
bựt
bựt
bựt
bựt
bựt
bốp
bốp
bốp
bốp
bốp
bốp
bộp
bộp
bộp
bộp
bộp
bộp
hoắc
hoắc
hoắc
hoắc
hoắc
hoắc
hoặc
hoặc
hoặc
hoặc
hoặc
hoặc
bêu
bêu
bều
bều
bều
bều
bều
bều
bệu
bệu
bệu
bệu
bệu
bệu
giễu
giễu
giễu
giễu
giễu
giễu
hếu
hếu
hếu
hếu
hếu
hếu
lểu
lểu
lểu
lểu
lểu
lểu
bòn
bòn
bón
bón
bọn
bọn
chỏn
chỏn
nhõn
nhõn
đon
đon
đon
bà
bá
bã
bạ
bả
đa
đa
buýt
buýt
suỵt
suỵt
bơi
bơi
bới
bới
bới
bới
bới
bới
bời
bời
bời
bời
bời
bời
bởi
bởi
bởi
bởi
bởi
bởi
cỡi
cỡi
cỡi
cỡi
cỡi
cỡi
gợi
gợi
gợi
gợi
gợi
gợi
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
duềnh
huênh
huênh
huênh
bét
bét
bẹt
bẹt
béc
béc
khẹc
khẹc
bàm
bàm
bám
bám
chạm
chạm
cảm
cảm
hãm
hãm
đam
đam
đam
huỳnh
huỳnh
huỳnh
luýnh
luýnh
luýnh
bì
bí
bĩ
bỉ
bị
đi
đi
choàng
choàng
choàng
choáng
choáng
choáng
choạng
choạng
choạng
choảng
choảng
choảng
doãng
doãng
doãng
giếc
giếc
giếc
giếc
giếc
giếc
bòng
bòng
bòng
bóng
bóng
bóng
bõng
bõng
bõng
bọng
bọng
bọng
bỏng
bỏng
bỏng
đong
đong
đong
đong
bít
bít
bịt
bịt
hoạch
hoạch
hoạch
ngoách
ngoách
ngoách
bùng
bùng
bùng
búng
búng
búng
bụng
bụng
bụng
bủng
bủng
bủng
cũng
cũng
cũng
đung
đung
đung
đung
bạp
bạp
cáp
cáp
bết
bết
bết
bết
bết
bết
bệt
bệt
bệt
bệt
bệt
bệt
têt
têt
bơ
bớ
bớ
bờ
bờ
bở
bở
bỡ
bỡ
bợ
bợ
bùm
bùm
bũm
bũm
bụm
bụm
bủm
bủm
chúm
chúm
đum
đum
đum
bìa
bìa
bịa
bịa
chĩa
chĩa
khía
khía
mỉa
mỉa
bân
bân
bấn
bấn
bấn
bấn
bấn
bấn
bần
bần
bần
bần
bần
bần
bẩn
bẩn
bẩn
bẩn
bẩn
bẩn
bận
bận
bận
bận
bận
bận
cẫn
cẫn
cẫn
cẫn
cẫn
cẫn
bích
bích
bích
bịch
bịch
bịch
buôn
buôn
buồn
buồn
buồn
buồn
buồn
buồn
cuốn
cuốn
cuốn
cuốn
cuốn
cuốn
cuộn
cuộn
cuộn
cuộn
cuộn
cuộn
thuỗn
thuỗn
thuỗn
thuỗn
thuỗn
thuỗn
bớp
bớp
bớp
bớp
bớp
bớp
bợp
bợp
bợp
bợp
bợp
bợp
bướp
bướp
bướp
bướp
bướp
bướp
bướp
bướp
bướp
bướp
bướp
bướp
chượp
chượp
chượp
chượp
chượp
chượp
chượp
chượp
chượp
chượp
chượp
chượp
bùa
bùa
búa
búa
bụa
bụa
bủa
bủa
giũa
giũa
đua
đua
đua
chưn
chưn
sừn
sừn
sừn
sừn
sừn
sừn
bèm
bèm
bẻm
bẻm
chém
chém
hẹm
hẹm
kẽm
kẽm
đem
đem
đem
bệch
bệch
bệch
bệch
bệch
bệch
bệch
bệch
bệch
bệch
bệch
bệch
chếch
chếch
chếch
chếch
chếch
chếch
chếch
chếch
chếch
chếch
chếch
chếch
bắp
bắp
bắp
bắp
bắp
bắp
bặp
bặp
bặp
bặp
bặp
bặp
bóc
bóc
bọc
bọc